content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Penglolaan Objek Kubernetes Secara Imperatif dengan Menggunakan File Konfigurasi
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Objek-objek Kubernetes bisa dibuat, diperbarui, dan dihapus dengan menggunakan perangkat *command-line* `kubectl` dan file konfigurasi objek yang ditulis dalam format YAML atau JSON. Dokumen ini menjelaskan cara mendefinisikan dan mengelola objek dengan menggunakan file konfigurasi.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Kelebihan dan kekurangan
+
+Perintah `kubectl` mendukung tiga cara pengelolaan objek:
+
+* Perintah imperatif
+* Konfigurasi objek imperatif
+* Konfigurasi objek deklaratif
+
+Lihat [Pengelolaan Objek Kubernetes](/id/docs/concepts/overview/object-management-kubectl/overview/) untuk mengenali lebih lanjut kelebihan dan kekurangan dari tiap cara pengelolaan objek.
+
+## Cara membuat objek
+
+Kamu bisa menggunakan perintah `kubectl create -f` untuk membuat sebuah objek dari sebuah file konfigurasi. Rujuk dokumen [referensi API Kubernetes](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/) untuk detil lebih lanjut.
+
+- `kubectl create -f <filename|url>`
```

Comment:
“filename” in English; is this correct?

Reply:
hi @sftim, you're right, I missed translating that part. I have updated it in my new commit.